src/CHANGELOG.tsx

```diff
@@ -17,12 +17,14 @@ import {
   Vireve,
   Pilsung,
   HerzBlutRaffy,
-  Abelito75
+  Abelito75,
+  Elitesparkle
 } from 'CONTRIBUTORS';
 import { ItemLink } from 'interface';
 import SpellLink from 'interface/SpellLink';
 
 export default [
+  change(date(2023, 5, 8), 'Fix Mana Costs for Shaman Spells and Talents.', Elitesparkle),
```

Comment:
This is the wrong file for spec-specific changes, you want to add your changelog entry at `src\analysis\retail\shaman\restoration\CHANGELOG.tsx`

Reply:
I went for that because I've changed the Mana Cost for Generic Spells and Talents too. Still wrong?

Reply:
yep